README.md

```diff
@@ -122,7 +122,7 @@ Returns true if the list of arguments in the **current function** matches the ty
 This framework is a work in progress.
 
 Mail me at spark.crz(at)gmail.com if you wanna chat!
-Or join `##php` and `##php-br` channels at [freenode.net IRC servers][1]
```

Comment:
The non-official channels of freenode have two hashes, that means the ##php channel doesn't belong to the php project team, but belongs to fans and/or the community members. As for ##php-br, there's no project named php-br so it's very unlikely they will ever loose the double hash.